language-design.md

```diff
@@ -59,7 +193,34 @@ What does the post on grayscale tell us about the process of API design?
 
 **Response**
 
-
+When designing an API, one should have a specific target user-base in mind so
```

Comment:
I think having a specific target user-base in mind is extremely important. As you mention below, it would be of no use to get people who don't understand rgb values to weigh in on the API naming decision.  Something that interested me was whether it mattered or not to get input from people who _will_ or _do_ use your product versus people with expertise or understanding of the subject matter.  Ultimately, it's the people who use your product that matter most, so should you give more weight to their opinions to make it intuitive to them, or make it as intuitive as possible to the general audience?